OpenTibia.Server.Events.MoveUseFile/MoveUseScriptApiAdapter.cs

```diff
@@ -0,0 +1,358 @@
+﻿// -----------------------------------------------------------------
+// <copyright file="MoveUseScriptApiAdapter.cs" company="2Dudes">
+// Copyright (c) 2018 2Dudes. All rights reserved.
+// Author: Jose L. Nunez de Caceres
+// http://linkedin.com/in/jlnunez89
+//
+// Licensed under the MIT license.
+// See LICENSE file in the project root for full license information.
+// </copyright>
+// -----------------------------------------------------------------
+
+namespace OpenTibia.Server.Events.MoveUseFile
+{
+    using System;
+    using OpenTibia.Common.Utilities;
+    using OpenTibia.Server.Contracts.Abstractions;
+    using OpenTibia.Server.Contracts.Enumerations;
+    using OpenTibia.Server.Contracts.Structs;
+    using OpenTibia.Server.Parsing.Contracts.Enumerations;
+    using Serilog;
+
+    /// <summary>
+    /// Class that represents an adapter between the Move/Use scripts and the current <see cref="IScriptApi"/> implementation.
+    /// </summary>
+    /// <remarks>
+    /// Names of functions in this class must match those found within the Move/Use file.
+    /// </remarks>
+    public class MoveUseScriptApiAdapter
```

Comment:
missing a lot of docs, or track on #23... but given the importance of this class, better document right now...

Reply:
doc'd